packages/components/src/components/link/link.tsx

```diff
@@ -0,0 +1,52 @@
+import { Component, h, Prop } from '@stencil/core';
+import { CssClassMap } from '../../utils/utils';
+import classNames from 'classnames';
+
+@Component({
+  tag: 't-link',
+  styleUrl: 'link.css',
+  shadow: true,
+})
+export class Link {
+  /** (optional) link href */
+  @Prop() public href?: string = '';
+  /** (optional) link disabled */
+  @Prop() public disabled?: boolean = false;
+  /** (optional) link variant */
+  @Prop() public underline?: boolean = false;
+  /** (optional) link open a new tag */
+  @Prop() public newTag?: boolean = false;
+  /** (optional) link variant */
+  @Prop() public variant?: string = '';
+
+  public render() {
+    if (!!this.href) {
+      if (!!this.newTag) {
+        return (
+          <a href={this.href} class={this.getCssClassMap()} target="_blank">
+            <slot />
+          </a>
+        );
+      }
+      return (
+        <a href={this.href} class={this.getCssClassMap()}>
```

Comment:
you need to check if the link is not disabled, and only then provide the `href` e.g.: `href={!this.disabled && this.href}`, otherwise the disabled link is still clickable

Reply:
i set css class -> cursor not allowed.  The link is not clickable

Reply:
it actually is :)